### ISSUE TITLE
How to wake over BLE

### ISSUE THREAD

**Issue by Andreas-lang** [teslamotors/vehicle-command#270]
I'm trying to wake the car over bluetooth, like this:

`tesla-control -ble wake`

I get one of two errors:

`Failed to execute command: MESSAGEFAULT_ERROR_INSUFFICIENT_PRIVILEGES
`

or

`Error: context deadline exceeded
`

Is it possible to wake over bluetooth? If yes, how did you do it?

System is MacOS 14.5.

**Comment by erikgieseler** [comment]
See https://github.com/teslamotors/vehicle-command/issues/248

**Comment by Andreas-lang** [comment]
Thank you Erik!

To get around `Error: context deadline exceeded`, per #248, it was important to include the parameter `-domain vcsec` in the call.

To get around the `Failed to execute command: MESSAGEFAULT_ERROR_INSUFFICIENT_PRIVILEGES`, I needed to ensure I was using the right private key. My mistake was that I forgot that the tesla-keygen utility stored the private key in my login keychain, and I needed to set the right environment variable so it could find it again.

As a side-note, how do I export the private key from the Mac OS keychain? I would prefer to have a file with a copy, just in case.
